### PR TITLE
Update webhook events

### PR DESCRIPTION
**Pull request by karolkielecki**

### CODE REVIEW

File: docs/dashboard/apps.md [hunk 1]
```diff
@@ -64,7 +64,17 @@ To learn more about the webhook protocols, see the [Developer's Guide](developer
 
 ### Events
 
-In the _Events_ card, you can assign singular events that you want to be triggered by the webhook. Alternatively, you can choose the _All events_ option to send notifications about all events, regardless of type. By choosing this option you will send payloads for all events currently assigned, in doing so your range will change as events are added or deleted by the Saleor team. If you want to assign a limited range of events, choose them from the menu of specific items.
+:::note
+Asynchronous and synchronous webhook events are available in Saleor 3.1 (alpha stage).
```
Comment (korycins) [inline docs/dashboard/apps.md:68]: IMO, Do we really need `(alpha stage)` here? I would just write Saleor 3.1.

File: docs/dashboard/apps.md [hunk 2]
```diff
@@ -64,7 +64,17 @@ To learn more about the webhook protocols, see the [Developer's Guide](developer
...
+:::
+
+In the *Events* card, you can assign singular events that you want to be triggered by the webhook.
+
+There are two types of events that can be assigned to a webhook: [synchronous](https://docs.saleor.io/docs/3.0/developer/extending/apps/synchronous-webhooks) and [asynchronous](https://docs.saleor.io/docs/3.0/developer/extending/apps/asynchronous-webhooks). 
+
+Payment webhooks are synchronous webhooks that allow delegating payment handling to Saleor Apps. You can select multiple synchronous events.
```
Comment (korycins) [inline docs/dashboard/apps.md:75]: Maybe I don't understand it well, but I assumed after reading this line, that synchronous webhooks means payments webhooks. (Maybe we could add a single line about sync webhooks similar to line for Asynchronous webhook)
Payment webhooks are a part of synchronous webhooks. A synchronous webhook can be a payment webhook but it also can be something different.

File: docs/dashboard/apps.md (hunk 1 above)
Comment (karolkielecki) [reply docs/dashboard/apps.md]: Removed

Comment (patrys) [reply docs/dashboard/apps.md]: Asynchronous events are available in previous versions of Saleor, it's synchronous callbacks that were added in 3.1. Can we make it more clear?

File: docs/dashboard/apps.md [hunk 3]
```diff
@@ -63,8 +63,17 @@ To learn more about the webhook protocols, see the [Developer's Guide](developer
...
+There are two types of events that can be assigned to a webhook: [synchronous](https://docs.saleor.io/docs/3.0/developer/extending/apps/synchronous-webhooks) and [asynchronous](https://docs.saleor.io/docs/3.0/developer/extending/apps/asynchronous-webhooks).
+
+Synchronous webhooks can be used for extending Saleor without the need to deploy plugins on the API server. You can select multiple synchronous events.
+
+Asynchronous webhook events can be used to receive data from Saleor when particular events happen. Alternatively, you can choose the Any events option. By choosing this option you will receive webhooks when any of the available events are triggered.
+
+Saleor 3.0 introduces synchronous payment webhooks that allow delegating payment handling to Saleor Apps. It also introduces asynchronous webhooks dedicated to filtering shipping methods in checkout and order.
```
Comment (korycins) [inline docs/dashboard/apps.md:74]: Is it expected that for 3.1 we provide exact name of the webhook, but for 3.0 we just leave a description?

**Review by korycins — APPROVED** [review]

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**